Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -532,8 +532,10 @@ def audit_conflicts
       end
     end
 
+    versioned_conflicts_whitelist = %w[node@ bash-completion@].freeze
```

Comment:
Yes, I'd leave this as is for now, and convert all of them in one go in a separate PR.